### PR TITLE
Update copyright headers to use 'The kpt Authors' instead of 'Google LLC'

### PR DESCRIPTION
**Pull request by mortent**
This updates the copyright headers in all files as well as the boilerplate files to use 'The kpt Authors' rather than 'Google LLC'. This does not change any copyright claims by contributors.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -115,7 +115,7 @@ test-docker: build
 # KPT_FN_RUNTIME can be set to select the desired function runtime.
 # If unspecified, the default function runtime will be used.
 test-fn-render: build
-	PATH=$(GOBIN):$(PATH) go test -v --tags=docker --run=TestFnRender/testdata/fn-render/$(T) ./e2e/
+	PATH=$(GOBIN):$(PATH) go test -v --tags=docker --run=TestFnSink/testdata/fn-sink/$(T) ./e2e/
```
Comment (natasha41575) [inline Makefile:118]: Why make this change? It doesn't look correct. It seems like we should have a separate make target `make test-fn-sink` to run the TestFnSink, and leave the `test-fn-render` target as is?

**Review by natasha41575 — COMMENTED** [review]
My browser had a hard time loading the diff 😅

Just one comment.

File: Makefile (hunk 1 above)
Comment (mortent) [reply Makefile:118]: Oh, good catch. I did not intend to commit this, it was just something I used for debugging. I will fix it.

Comment (mortent) [reply Makefile]: Fixed now.

**Review by natasha41575 — APPROVED** [review]

**Pull request merged**